core/workspace_svg.js

```diff
@@ -1149,6 +1149,8 @@ Blockly.WorkspaceSvg.prototype.onMouseWheel_ = function(e) {
  */
 Blockly.WorkspaceSvg.prototype.getBlocksBoundingBox = function() {
   var topBlocks = this.getTopBlocks(false);
+  var topComments = this.getTopComments(false);
+  topBlocks = topBlocks.concat(topComments);
```

Comment:
Let's call this `topElements` instead.  Calling it `topBlocks` implies things that aren't true.